lib/AST/NameLookup.cpp

```diff
@@ -1946,7 +1946,33 @@ static DirectlyReferencedTypeDecls
 directReferencesForUnqualifiedTypeLookup(DeclNameRef name,
                                          SourceLoc loc, DeclContext *dc,
```

Comment:
Seems a bit odd that we still pass in a DC here, esp. since passing in both a location and a DC.

Reply:
I'm going to do a separate PR that redoes the user-facing unqualified lookup entry point to take a SourceFile instead of a DC. The first step was making sure the DC wasn't actually used internally :)

Reply:
Sounds good!